src/main/java/hudson/plugins/fitnesse/NativePageCounts.java

```diff
@@ -59,7 +61,7 @@ public void startElement(String uri, String localName, String qName, Attributes
 		if (COUNTABLE.contains(qName)) {
 			String targetPage;
 			if (qName.equals(SUMMARY)) {
-				targetPage = "Summary";
+				targetPage = resultFileName;
 			} else {
 				String page = attributes.getValue(PAGE);
 				String pseudoPage = attributes.getValue(PSEUDO_PAGE);
```

Comment:
this commit concerne the bug of JENKINS-27936, please do 2 differents commits, and 2 differents pull requests